openlibrary/templates/home/index.html

```diff
@@ -40,7 +40,8 @@
 
   $:render_template("home/categories", test=test)
 
-  $:render_template("home/stats", stats)
+  $if not hide_stats:
+    $:render_template("home/stats", stats)
```

Comment:
Several elements in [`home/stats`](https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/home/stats.html) source data from IA.  It seems cleaner to just remove the component while in offline-mode.